dom.bs

```diff
@@ -1488,7 +1488,7 @@ can be used to explore this matter in more detail.
 
 <h3 id=node-trees>Node tree</h3>
 
-<p>{{Document}}, {{DocumentType}}, {{DocumentFragment}}, {{Element}}, {{Text}},
+<p>{{Document}}, {{DocumentType}}, {{DocumentFragment}}, {{Element}}, {{Text}}, {{CDATASection}},
```

Comment:
OK, I guess it's "So the new policy is to list concrete classes (DocumentFragment) unless they’re a subclass of a concrete class (ShadowRoot) and not list abstract classes (CharacterData)."